README.md

```diff
@@ -23,145 +23,165 @@ which functions as the data backbone for prominent hedge funds, banks, and finan
 
 | Language      | Server Application | Client Application (OpenAPI) |
 | ------------- | ------------------ | ---------------------------- |
-| Python        | Yes                | No                           |
-| Java / Groovy | Yes                | No                           |
+| Python        | Yes                | Yes                           |
+| Java / Groovy | Yes                | Yes                           |
 | JavaScript    | No                 | Yes                          |
 | gRPC          | -                  | Yes                          |
 
-## Running Deephaven
+## Run Deephaven
+
+This section is a quick start guide for running Deephaven from pre-built images.
+For detailed instructions on running Deephaven, see [Launch Deephaven from pre-built images](https://deephaven.io/core/docs/tutorials/launch-pre-built).
+
+For detailed instructions on building and running Deephaven from source code, 
+see [Build and launch Deephaven](https://deephaven.io/core/docs/tutorials/launch-build).
```

Comment:
```suggestion
see [Build and launch Deephaven](https://deephaven.io/core/docs/tutorials/launch-build). Building from source is for developers that are modifying the Deephaven Community Core source code.

```

Reply:
"see" needs to join the line above - I'm unable to remove that line break in suggestion mode.

Reply:
The structure would be better to have the intended audience followed by the link.

```suggestion
Developers interested in tinkering with and modifying source code should build from the source code.  For detailed instructions on how to do this, see [Build and launch Deephaven](https://deephaven.io/core/docs/tutorials/launch-build).
```